site/content/docs/developing/domain.en.md

```diff
@@ -1,13 +1,23 @@
 # Domain
 
 !!!attention
-    Today, a Route 53 domain name can be associated only when running `copilot app init`.  
-    If you'd like to update your application with a domain ([#3045](https://github.com/aws/copilot-cli/issues/3045)), 
-    you'll need to initialize a duplicate app with `--domain` and then run `copilot app delete` to 
-    remove the old one.
+    Today, a Route 53 domain name can only be associated when running `copilot app init`.  
+    If you'd like to update your application with a domain ([#3045](https://github.com/aws/copilot-cli/issues/3045)),
+    you'll need to run `copilot app delete` to remove the old one before creating a new one with `--domain` to associate a new domain.
```

Comment:
Should we move this attention block after your new bullet point:
> 1. Use `--domain` when creating an application to associate a Route 53 domain in the same account.

This way they can see that there is a second option:
> 2. Use `--import-cert-arns` to bring your validated ACM certificates when creating an environment. 

and don't need to re-create the app